components/GalleryPictures.tsx

```diff
@@ -6,7 +6,7 @@ type Props = {};
 
 function GalleryPictures({}: Props) {
   return (
-    <article>
+    <article className="flex flex-col rounded-lg items-center space-y-7 flex-shrink-0 w-[500px] md:w-[600px] xl:w-[900px] snap-center bg-[#292929] p-10 hover:opacity-100 opacity-40 cursor-pointer transition-opacity duration-200 overflow-hidden">
```

Comment:
why is this an `article`?

Reply:
for the purpose of SEO later on down the road since each of these will be a display of work done or possible work to try and get attention to their business